.github/workflows/sync-labels.yml

```diff
@@ -11,7 +11,8 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: micnncim/action-label-syncer@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           manifest: .github/labels.yml
+        repository: ${{ RESEARCH_LABEL_SYNC_REPOS }}
```

Comment:
ah clever. Do note that doing a commit does trigger the sync flow. If you don't do commit to change the secret then you'll need to re trigger the flow somehow